### ISSUE TITLE
Splitter component is not keyboard-accessible

### ISSUE THREAD

**Issue by pbb72** [quasarframework/quasar#12466]
### What happened?

Keyboard users are not able to control the splitter component.

### What did you expect to happen?

Using the tab key to navigator through the document, the splitter component should receive focus at the appropriate moment. This focus state should in some way be visible. When the splitter has focus, it's position should be controllable using the keyboard, for example using arrow keys.

### Reproduction URL

https://codepen.io/pbb/pen/bGYryEB

### How to reproduce?

1. Go to reproduction URL
2. Place focus on the "Before" panel by clicking the mouse in it.
3. Press tab to navigate to the next focusable element.
4. Notice that the splitter is not getting focus.

### Flavour

Quasar CLI (@quasar/cli | @quasar/app)

### Areas

Quasar CLI Commands/Configuration (@quasar/cli | @quasar/app)

**Comment by rstoenescu** [comment]
Fix will be available in v2.25
